api/src/main/thrift/org/apache/aurora/gen/api.thrift

```diff
@@ -197,6 +197,15 @@ struct DockerParameter {
   2: string value
 }
 
+struct Variable {
```

Comment:
+1. Maybe be more verbose with the type? InstanceParameterizationVariable ? (maybe I overreacted)